docs/docs/hardware.mdx

```diff
@@ -41,25 +41,19 @@ export const toc = [
 ];
 
 With the solid technical foundation of Zephyr™ RTOS, ZMK can support a wide diversity of hardware targets.
-That being said, there are currently only a few specific [boards](/docs/faq#what-is-a-board)/[shields](faq.md#what-is-a-shield) that have been implemented and tested by the ZMK contributors.
+That being said, there are specific [boards](/docs/faq#what-is-a-board)/[shields](faq.md#what-is-a-shield) that have been implemented and tested by the ZMK contributors, listed below.
```

Comment:
These two links go to the same page, but one is absolute and the other is relative. Would be nice to make that consistent.

Reply:
Good catch, thanks.